geode-web-api/src/main/java/org/apache/geode/rest/internal/web/controllers/BaseControllerAdvice.java

```diff
@@ -50,6 +52,11 @@
 
   protected static final String REST_API_VERSION = "/v1";
 
+  public BaseControllerAdvice(final RestSecurityService securityService,
```

Comment:
BaseControllerAdvice extends AbstractBaseController.  Since the RestSecurityService lives in AbstractBaseController, yes.  Otherwise, we define separate member variables in PdxBasedCrudController, CommonCrudController, and QueryAccessContoller.